### PR TITLE
Translations/merge library translations

### PR DESCRIPTION
**Pull request by FrozenKiwi**
Closes https://github.com/thecointech/thecoin/issues/215

Merge library translation components into output apps.

Tested in landing/app/NFT by manually checking that at least 1 string from library text was translated properly: eg "Add Account"

image

### CODE REVIEW

File: apps/site-landing/src/containers/MainNavigation/index.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const menuItems = defineMessages({
   help: {
     defaultMessage: 'Help',
     description: 'site.MainNavigation.faq: Help Link',
-    icon: "question mark"
+    icon: "question"
```
Comment (FrozenKiwi) [inline apps/site-landing/src/containers/MainNavigation/index.tsx:30]: This icon worked but threw an "Unknown name" in debug mode.

File: libs/shared/lang/en.json [hunk 2]
```diff
@@ -1,182 +1,202 @@
...
-  "024I2T": {
-    "defaultMessage": "Balance",
-    "description": "shared.widgets.balanceandprofit.balance: Title for widget Balance and profit"
-  },
-  "0zfmOG": {
-    "defaultMessage": "Cancel",
-    "description": "shared.modalOperation.cancel: Text for the cancel button for the modal tool"
-  },
-  "2Rp4o1": {
+  "'+yCmHT'": {
```
Comment (FrozenKiwi) [inline libs/shared/lang/en.json:2]: This language file is a compiled output, so probably could be removed (perhaps when reviewing translation pipeline)

File: libs/shared/src/containers/NotFoundPage/index.tsx [hunk 3]
```diff
@@ -10,17 +10,14 @@ import { Header, Grid, Button } from 'semantic-ui-react';
 import { NavLink } from 'react-router-dom';
 import illustration from './images/illust_pagenotfound.svg'
 
-const translate = defineMessages({ 
-  aboveTheTitle : { 
-    id: "shared.NotFoundPage.aboveTheTitle",
```
Comment (FrozenKiwi) [inline libs/shared/src/containers/NotFoundPage/index.tsx:15]: for some reason, in-built id's were not found when importing this message

File: apps/site-landing/src/translations/index.ts [hunk 4]
```diff
@@ -1,7 +1,18 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
 import { formatTranslations } from "@thecointech/shared/containers/LanguageProvider/initialize"
+import en from './en.json';
+import fr from './en.json';
```
Comment (MarieTheCoin) [inline apps/site-landing/src/translations/index.ts:3]: Is this right?

File: apps/site-nft/src/translations/index.ts [hunk 5]
```diff
@@ -1,7 +1,18 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
 import { formatTranslations } from "@thecointech/shared/containers/LanguageProvider/initialize"
+import en from './en.json';
+import fr from './en.json';
```
Comment (MarieTheCoin) [inline apps/site-nft/src/translations/index.ts:3]: Same here?

File: libs/shared/lang/en.json [hunk 6]
```diff
@@ -1,182 +1,202 @@
...
   },
-  "iAyWom": {
-    "defaultMessage": "Log into",
-    "description": "shared.login.title: Title for the create account form page"
+  "'y+kks9'": {
+    "defaultMessage": "strong",
+    "description": "Password complexity indicator"
   },
-  "jNMJW6": {
+  "shared.account.uploadWallet.aboveTheTitle": {
```
Comment (MarieTheCoin) [inline libs/shared/lang/en.json:110]: Is this normal?

File: apps/site-landing/src/translations/index.ts (hunk 4 above)
Comment (FrozenKiwi) [reply apps/site-landing/src/translations/index.ts:3]: LOL - hella no!  Great catch Marie!!!

File: apps/site-nft/src/translations/index.ts (hunk 5 above)
Comment (FrozenKiwi) [reply apps/site-nft/src/translations/index.ts:3]: yikes... (fixed)

**Pull request merged**